app/handlers/base.py

```diff
@@ -392,3 +392,9 @@ def push_notification(self, note, notification_type='info'):
             action='baselayer/SHOW_NOTIFICATION',
             payload={'note': note, 'type': notification_type},
         )
+
+    def get_query_argument(self, name, default=RequestHandler._ARG_DEFAULT, strip=True):
+        ret = RequestHandler.get_query_argument(self, name, default, strip)
+        if type(default) == bool:
+            ret = type(ret) == str and ret.lower() in ['true', 'yes', 't', '1']
```

Comment:
```suggestion
            return arg.lower() in ['true', 'yes', 't', '1']
```

Reply:
What if `arg` is None?

Reply:
Yes, that can happen. How about `str(arg).lower() in ...`.